services/checkin/service/auth_service.go

```diff
@@ -26,3 +26,21 @@ func AddAttendeeRole(id string) error {
 
 	return nil
 }
+
+/*
+	Gets the roles for a user given id.
+*/
+func GetRoles(id string) (*models.UserRoles, error) {
+	var user_roles models.UserRoles
+	status, err := apirequest.Get(config.AUTH_SERVICE+"/auth/roles/"+id+"/", user_roles)
```

Comment:
Did you test this? You should have to pass `user_roles` as a pointer. So `&user_roles`.